pkg/pillar/types/zedmanagertypes.go

```diff
@@ -67,6 +67,8 @@ type AppInstanceConfig struct {
 	PurgeCmd            AppInstanceOpsCmd
 	CloudInitUserData   *string // base64-encoded
 	RemoteConsole       bool
+	IsCipher            bool
```

Comment:
Please add comment stating that when IsCipher is set, the the CipherBlock contains the CloudInitUserData (in which field??) and the above CloudInitUserData is empty/ignored.

Reply:
sure, will do